src/js/index.js

```diff
@@ -1,12 +1,31 @@
 var React = require('react');
 var ReactDOM = require('react-dom');
-var makeSubmitForm = require('./makeSubmitForm.jsx');
+var api = require('./api');
+var userSelect = require('./userSelect.jsx');
+var fiContainer = require('./fiContainer.jsx');
+//var makeSubmitForm = require('./makeSubmitForm.jsx');
 
+/*
 var SubmitForm = makeSubmitForm('/submit', function(e){
   console.log(e, 'file uploaded');
 });
+*/
 
 ReactDOM.render(
```

Comment:
Well actually, I think we will need it for production, just probably not in index.js. That's the submit component aye?
We'd just need to route to it and it probably/possibly won't be included here.

I'll nix it from index.js